gsa/src/web/store/entities/utils/__tests__/actions.js

```diff
@@ -662,4 +663,33 @@ describe('entities actions tests', () => {
     });
   });
 });
+
+describe('createDeleteEntity tests', () => {
+  test('should create a entity delete success action', () => {
+    const id = 'id1';
+    const gmp = {
+      foo: {
+        delete: jest.fn().mockReturnValue(
+          Promise.resolve({
```

Comment:
This can be simplified by using [jest.fn().mockResolvedValue({data: 'ba'});](https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvaluevalue)